roles/common/tasks/debian.yml

```diff
@@ -252,15 +252,33 @@
       - python3-pip
     state: present
   become: true
-  tags: package
+  tags:
+    - package
+    - pip-package
+  when: ansible_distribution_major_version != "12"
 
 - name: Upgrade pip
   ansible.builtin.pip:
     name: pip
     extra_args: --upgrade
-  tags: package
+  tags:
+    - package
+    - pip-package
+  when: ansible_distribution_major_version != "12"
 
 - name: Install pip packages
   ansible.builtin.pip:
     name: "{{pip_packages}}"
+  tags:
+    - package
+    - pip-package
+  when: ansible_distribution_major_version != "12"
+
+- name: Install pip packages using apt
+  apt:
+    name:
+      - python3-cryptography
+    state: present
+  become: true
```

Comment:
why don't we have pip-package tag here?

Reply:
pip-package tag is added because in newer versions of python we are getting errors in the tasks where we run pip install. So if somehow that kind of issues occur then to skip those task pip-package can  be used in --skip-tags. This on the other hand is apt install which wont be causing failures

Reply:
have we added the tag in ubuntu.yml file? I can't see those changes in this pr

Reply:
I thgouht since we are not onboarding newer ubuntu versions this wont be needed in ubuntu.yml. but it does no harm even if have may come in handy someday. So will add it there as well

Reply:
it's not just about new versions, customers will start using this tag to skip all pip tasks, so this tag needs to be present in all tasks which are doing pip installation